snippet-extractor/Pipfile

```diff
@@ -0,0 +1,14 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+langchain = "*"
+transformers = "*"
+nltk = "*"
```

Comment:
The versions are fixed in the lock file.

Also because python was very bad at versioning until very recently it was somewhat rare to ever do a new major release after 1.0. 

You also can't have two different versions of the same library which makes depending on concrete version much more tricky.

So we basically want the newest compatible version of the combinations of above's library.

Through I guess we should set the minimal version to the currently locked version.